Lib9c/Module/GiftModule.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Bencodex.Types;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Nekoyume.Model.State;
+
+namespace Nekoyume.Module
+{
+    public static class GiftModule
+    {
+        public static List<int> GetClaimedGifts(this IWorldState state, Address avatarAddress)
+        {
+            var account = state.GetAccountState(Addresses.ClaimedGiftIds);
+            return account.GetState(avatarAddress) is List rawIds
+                ? rawIds.ToList(serialized => (int)(Integer)serialized)
+                : new List<int>();
+        }
+
+        public static IWorld SetClaimedGifts(this IWorld world, Address avatarAddress, List<int> claimedGiftIds)
+        {
+            var account = world.GetAccount(Addresses.ClaimedGiftIds);
+            account = account.SetState(avatarAddress, claimedGiftIds.Aggregate(List.Empty, (current, giftId) => current.Add((Integer)giftId)));
+            return world.SetAccount(Addresses.ClaimedGiftIds, account);
```

Comment:
`MutateAccount` 를 활용하면 위 `GetAccount` → `SetAccount` 하는 과정을 쉽게 할 수 있습니다.

```suggestion
            return world.MutateAccount(
                Addresses.ClaimedGiftIds,
                account => account.SetState(
                    avatarAddress,
                    claimedGiftIds.Aggregate(List.Empty, (current, giftId) => current.Add((Integer)giftId))
                )
            );
```